components/automate-ui/src/app/entities/reset-admin-key/reset-admin-key.actions.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Action } from '@ngrx/store';
+
+import { AdminKey } from './reset-admin-key.model';
+
+export enum AdminKeyActionTypes {
+  UPDATE = 'ADMINKEY::UPDATE',
+  UPDATE_SUCCESS = 'ADMINKEY::UPDATE::SUCCESS',
+  UPDATE_FAILURE = 'ADMINKEY::UPDATE::FAILURE'
+}
+
+export interface AdminKeySuccessPayload {
+  adminKey: AdminKey;
+}
+
+export class UpdateAdminKey implements Action {
+  readonly type = AdminKeyActionTypes.UPDATE;
+
+  constructor(public payload: {server_id: string, org_id: string, admin_Key: AdminKey} ) { }
```

Comment:
Please use a named type instead of inline, perhaps:
```suggestion
  constructor(public payload: UpdateAdminKeyPayload ) { }
```
Also, please change the `admin_Key` field to `admin_key`.